src/extension.ts

```diff
@@ -73,15 +73,17 @@ import { Logging } from './shared/logger/commands'
 import { showMessageWithUrl, showViewLogsMessage } from './shared/utilities/messages'
 import { registerWebviewErrorHandler } from './webviews/server'
 import { initializeManifestPaths } from './extensionShared'
+import { ChildProcess } from './shared/utilities/childProcess'
 
 let localize: nls.LocalizeFunc
 
 export async function activate(context: vscode.ExtensionContext) {
     await initializeComputeRegion()
     const activationStartedOn = Date.now()
     localize = nls.loadMessageBundle()
-    initialize(context)
-    initializeManifestPaths(context)
+
+    await initialize(context)
+    ;(globals.machineId = await getMachineId()), initializeManifestPaths(context)
```

Comment:
Whats the reason for this syntax? Couldn't it just be
```suggestion
    await initialize(context)
    initializeManifestPaths(context)
    globals.machineId = await getMachineId()
```

Reply:
I think the linter put this in? Let me try swapping it.

Reply:
Yeah IDK, maybe it was just something weird, it went through this time.